server/src/immich/controllers/person.controller.ts

```diff
@@ -44,7 +48,12 @@ export class PersonController {
     return this.service.update(authUser, id, dto);
   }
 
-  @Get(':id/thumbnail')
+  @Get('/count')
+  getPersonCount(@AuthUser() authUser: AuthUserDto): Promise<PersonCountResponseDto> {
```

Comment:
Why do we need this endpoint?

Reply:
On the explore page, the server sends back only unhidden faces; however, to determine whether to display the 'view all' button, it needs to know the total count of both hidden and unhidden faces.

Reply:
Could you also just return with hidden and filter them out?